cranelift-simplejit/src/backend.rs

```diff
@@ -177,7 +183,7 @@ impl<'simple_jit_backend> Backend for SimpleJITBackend {
         let size = code_size as usize;
         let ptr = self
             .code_memory
-            .allocate(size)
+            .allocate(size, 0x10)
```

Comment:
This is the same default as the one faerie uses.

Reply:
It's indeed the right constant for x64; for ARM it can be only 0x8, it seems. Being conservative and having 0x10 all the time sounds fine to me. Can you make this a global constant, please?

Reply:
Yes